app/models/manager_refresh/inventory_object.rb

```diff
@@ -44,10 +44,17 @@ def attributes(inventory_collection_scope = nil)
       data.each do |key, value|
         if !allowed?(inventory_collection_scope, key)
           next
-        elsif loadable?(value)
+        elsif value.kind_of?(Array) && value.any? { |x| loadable?(x) }
+          # Lets fill also the original data, so other InventoryObject referring to this attribute gets the right
+          # result
+          data[key]                                            = value.compact.map(&:load).compact
```

Comment:
I probably like aligned `=` more than most people :) but with two lines of comments between this and the next assignment I don't think it helps readability